src/pages/Publisher/Publisher.js

```diff
@@ -0,0 +1,82 @@
+import ToolHeader from '../../components/common/ToolHeader.js';
+import ToolFooter from '../../components/common/ToolFooter.js';
+import CapabilitiesRight from '../../components/common/CapabilitiesRight.js';
+import Description from '../../components/common/Description.js';
+
+const Publisher = () => (
+  <Box>
+    <AppHeader />
+    <Main>
+      <ToolHeader
+        mainIcon={<CloudUpload size="xxlarge" color="PublisherPink" />}
+        icon1={<Camera opacity="0.8" size="large" color="ImagerOrange" />}
+        icon2={
+          <ClearOption opacity="0.9" size="large" color="DesignerYellow" />
+        }
+        icon3={<Configure opacity="0.7" size="large" color="ThemerOrange" />}
+        icon4={<Selection opacity="0.5" size="large" color="SlidesBlue" />}
+        icon5={<BarChart opacity="0.3" size="large" color="TabularGreen" />}
+        title="Publisher"
+        content="Grommet Publisher is a tools used to build simple and beautiful documentation.
+        Quickly generate content, generate a themer, and share the site URL with anyone in just a few clicks"
+        open="Open Publisher"
+        link="https://publisher.grommet.io/"
+        color="PublisherPink"
+      />
+      <Description
+        title="Publisher"
+        content="A free, web-based documentation generator, the Publisher combines the
+        freedom of markup language with the efficiency of built-in text and button components.
+         See examples of the Publisher below."
+        color="PublisherPink"
+        images={['Designer_1.svg', 'Designer_2.svg', 'Designer_3.svg']}
+        labels={[
+          'Publisher File Name',
+          'Publisher File Name',
+          'Publisher File Name',
+        ]}
+      />
+
+      <CapabilitiesRight
+        mainContent="The Publisher provides a space to quickly generate and publish
+        content in minutes. Import a theme, logo, and add copyright details
+        directly to your page."
+        subContent="Just a few of the Publisher’s powerful capabilities
+        include using markup language to quickly create content, easily
+        importing and using a theme, and being able to publish and
+        share a URL with anyone."
+        mainImage="PublisherGremlin.svg"
+        mainLabel="Gremlin Holding a Publisher Icon"
+        images={['Publisher_1.svg', 'Designer_5.svg', 'Publisher_2.svg']}
+        labels={[
+          'Use Markup language to quickly generate content',
+          'Easily import and use theme',
+          'Publish and share URL with anyone in just a couple clicks',
+        ]}
+      />
+
+      <ToolFooter
+        backgroundColor="PublisherPink"
+        headingText="Start Using the Publisher Now"
+        description="Open the Publisher tool and get started by watching a tutorial or
+        just jumping right in to your first design."
+        buttonLabel="Open Publisher"
+        buttonHref="https://publisher.grommet.io"
+        direction="column"
```

Comment:
I didn't catch this earlier but do we need to pass in the direction prop for the footer? This shouldn't change on any of the pages right

Reply:
We need the direction to be set to `column` on the footer so that the gradient goes after the box with the button in it.

Reply:
Yes this is what it looks like without the direction set to column 
<img width="1704" alt="Screen Shot 2020-07-30 at 11 32 43 AM" src="https://user-images.githubusercontent.com/49867133/88942582-76c81300-d258-11ea-82c2-4e51c6c4f598.png">